backend/src/router_test.go

```diff
@@ -515,26 +515,28 @@ func (suite *RouterSuite) TestPatchProfile() {
 	}))
 }
 
-func (suite *RouterSuite) TestPutTutorsUsername() {
+func (suite *RouterSuite) TestPostTutors() {
 	suite.Run("Success", manualSetupTest(func() {
 		hash, _ := bcrypt.GenerateFromPassword([]byte("Password"), bcrypt.DefaultCost)
-		DB.Create(&User{Username: "Username", Password: string(hash)})
-
-		w := POST("/signin", gin.H{
+		
+		w := POST("/signup", gin.H{
 			"username": "Username",
-			"password": "Password",
+			"password": string(hash),
```

Comment:
This would just be `"Password"`, the hashing is performed server side.

Reply:
Fixed